src/pages/_app.js

```diff
@@ -0,0 +1,8 @@
+import "../styles/globals.scss"
+import '../styles/components/header.scss'
```

Comment:
We should be importing all of our component SCSS into `globals.scss`, rather than into our JSX files.